src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -501,7 +508,14 @@ std::shared_ptr<InputModel> InputModel::InputModelTFImpl::get_body_input_model(
     if (!body_graph_iterator) {
         return nullptr;
     }
-    return std::make_shared<InputModel>(body_graph_iterator, m_telemetry);
+    return std::make_shared<InputModel>(body_graph_iterator,
+                                        m_telemetry,
+                                        std::shared_ptr<ov::frontend::tensorflow::VariablesIndex>{},
+                                        nullptr,
+                                        nullptr,
+                                        nullptr,
+                                        false,
+                                        m_graph_iterator->tensor_names_need_indices());
```

Comment:
nit: I think let us leave it by default. For body graphs it is not critical at all.

Reply:
Done.